ProcessMaker/Policies/MediaPolicy.php

```diff
@@ -104,6 +104,15 @@ public function delete(User $user, Media $media)
             return true;
         }
 
-        return $user->can('update', $media->model);
+        if ($user->can('update', $media->model)) {
+            return $user->can('update', $media->model);    
+        }
+        
+        if($user->can('participate', $media->model)) {
+            return $user->can('participate', $media->model);
```

Comment:
@gustavobascope For this and the one above it, inside the conditional you can `return true;` instead of running the permission check again